Apps/W1/HybridGP/app/src/Migration/Support/HelperFunctions.codeunit.al

```diff
@@ -1274,6 +1275,10 @@ Codeunit 4037 "Helper Functions"
                 PostGLBatch(CopyStr(JournalBatchName, 1, 10));
         end;
 
+        OnSkipPostingPurchaseOrderBatches(SkipPosting);
```

Comment:
why changing the pattern used? If new pattern is better, why not change the others too?